### PR TITLE
Panos interface

### PR DESCRIPTION
**Pull request by ivanbojer**
##### ISSUE TYPE
 - New Module Pull Request

##### COMPONENT NAME
panos_interface

##### ANSIBLE VERSION

```
ansible 2.1.0.0 
config file = 
configured module search path = Default w/o overrides
```

##### SUMMARY
New panos module that allows Firewall interface configuration. Configure data-port (DP) network interface for DHCP.

### CODE REVIEW

**Comment by ivanbojer** [comment]
ready_for_review

File: lib/ansible/modules/network/panos/panos_interface.py [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+description:
+    - Configure data-port (DP) network interface for DHCP. By default DP interfaces are static.
+author: "Luigi Mori (@jtschichold), Ivan Bojer (@ivanbojer)"
+version_added: "2.3"
+requirements:
+    - pan-python
+options:
+    ip_address:
+        description:
+            - IP address (or hostname) of PAN-OS device
```
Comment (gundalow) [inline lib/ansible/modules/network/panos/panos_interface.py]: Capital letters & full stops please

File: lib/ansible/modules/network/panos/panos_interface.py [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+        default: "false"
+    commit:
+        description:
+            - commit if changed
+        required: false
+        default: true
+'''
+
+EXAMPLES = '''
+# enable DHCP client on ethernet1/1 in zone public
```
Comment (gundalow) [inline lib/ansible/modules/network/panos/panos_interface.py]: This comment should be part of the `- name:`

**Review by gundalow — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/network/panos/panos_interface.py (hunk 1 above)
Comment (ivanbojer) [reply lib/ansible/modules/network/panos/panos_interface.py]: OK

File: lib/ansible/modules/network/panos/panos_interface.py (hunk 2 above)
Comment (ivanbojer) [reply lib/ansible/modules/network/panos/panos_interface.py]: OK

**Comment by ivanbojer** [comment]
@gundalow Thank you for the review. I've incorporated all the changes requested. In addition I incorporated changes from the previous comments too (like ANSIBLE_METADATA coming before DOCUMENTATION).

**Comment by ivanbojer** [comment]
ready_for_review

**Review by gundalow — APPROVED** [review]

**Pull request merged**